### PR TITLE
Allow OpenSSH @9

### PR DESCRIPTION
**Pull request by eschnett**
Closes https://github.com/JuliaDocs/DocumenterTools.jl/issues/73

Supersedes https://github.com/eschnett/DocumenterTools.jl/pull/1

### CODE REVIEW

**Comment by eschnett** [comment]
I checked the output of `OpenSSH_jll.ssh_keygen` for the new version 9, and there doesn't seem to be any difference to previous versions.

**Comment by eschnett** [comment]
@mortenpi Can you approve the CI workflow?

**Comment by mortenpi** [comment]
LGTM. Could you also bump the patch version and add a brief note to `CHANGELOG.md`? We can then tag a new version right away.

**Comment by eschnett** [comment]
@mortenpi Done. Can you approve the new CI workflow?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # DocumenterTools.jl changelog
 
+## Version `v0.1.18`
+
+* ![Enhancement][badge-enhancement] Allow OpenSSH_jll 9 as dependency. ([#74][github-74])
```
Comment (mortenpi) [inline CHANGELOG.md:5]: Also needs a link reference down at the bottom.

Comment (eschnett) [reply CHANGELOG.md:5]: Done.

**Comment by eschnett** [comment]
@mortenpi Can you approve the CI workflow?

**Comment by mortenpi** [comment]
It looks like OpenSSH v9 doesn't work on Windows?

**Comment by eschnett** [comment]
This is difficult to debug for me since I don't have access to a Windows machine.

Given that there is no error message on `stdout` or `stderr`, I conjecture that `ssh-keygen.exe` did not actually start. This could point to a problem with shared libraries.

Maybe this https://github.com/JuliaPackaging/Yggdrasil/pull/7204 will correct the problem.

**Comment by giordano** [comment]
> This is difficult to debug for me since I don't have access to a Windows machine.

Usual reminder that https://github.com/mxschmitt/action-tmate is a thing.

For the record, the problem *may* be this missing `libcrypto.dll`
```julia-repl
julia> using DependencyWalker

julia> Library(OpenSSH_jll.ssh_keygen_path)
◼ C:\Users\runneradmin\.julia\artifacts\bed08e38e883f52775d130e1f289ee06485dbc6c\bin\ssh-keygen.exe
  ◼ C:\Windows\System32\CRYPT32.dll
  ✗ libcrypto.dll (NOT FOUND)
  ◼ C:\Windows\System32\USER32.dll
    ◼ C:\Windows\System32\GDI32.dll
    ◼ C:\Windows\System32\KERNELBASE.dll
  ◼ C:\Windows\System32\WS2_32.dll
    ◼ C:\Windows\System32\RPCRT4.dll
      ◼ C:\Windows\System32\KERNELBASE.dll
  ◼ C:\Windows\System32\KERNEL32.DLL
    ◼ C:\Windows\System32\KERNELBASE.dll
  ◼ C:\Windows\System32\ADVAPI32.dll
    ◼ C:\Windows\System32\KERNEL32.DLL
      ◼ C:\Windows\System32\KERNELBASE.dll
    ◼ C:\Windows\System32\RPCRT4.dll
      ◼ C:\Windows\System32\KERNELBASE.dll
    ◼ C:\Windows\System32\msvcrt.dll
      ◼ C:\Windows\System32\KERNELBASE.dll
    ◼ C:\Windows\System32\KERNELBASE.dll
    ◼ C:\Windows\System32\sechost.dll
      ◼ C:\Windows\System32\bcrypt.dll
```

**Comment by giordano** [comment]
https://github.com/JuliaPackaging/Yggdrasil/pull/9482 should fix the issue

File: Project.toml [hunk 2]
```diff
@@ -18,7 +18,7 @@ AbstractTrees = "0.3, 0.4"
 DocStringExtensions = "0.7, 0.8, 0.9"
 Documenter = "0.20, 0.21, 0.22, 0.23, 0.24, 0.25, 0.26, 0.27"
 Gumbo = "0.7, 0.8"
-OpenSSH_jll = "8"
+OpenSSH_jll = "8, 9"
```
Comment (giordano) [inline Project.toml:21]:
```suggestion
OpenSSH_jll = "8, 9.9.1"
```

File: Project.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name = "DocumenterTools"
 uuid = "35a29f4d-8980-5a13-9543-d66fff28ecb8"
-version = "0.1.17"
+version = "0.1.18"
```
Comment (giordano) [inline Project.toml:3]: I guess this is going to be
```suggestion
version = "0.1.20"
```
now.  Same in the ChangeLog above

**Comment by giordano** [comment]
I opened #91 to resolve the merge conflicts.

**Pull request closed without merging**